### PR TITLE
update README

### PR DESCRIPTION
**Pull request by MiaAltieri**
Now that [mongos is live](https://charmhub.io/mongos) lets update the README so that when users navigate to our repo they aren't met with empty information

README copied from [Overview of Charm](https://charmhub.io/mongos)

### CODE REVIEW

**Review by dmitry-ratushnyy — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+To see the Charmed MongoDB features and releases, visit our [Release Notes page](https://github.com/canonical/mongos-operator/releases). Currently the charm supports:
+- Automatic User Creation for application users
+
+## Charm version, environment and OS
+
+A charm version is a combination of both the application version and / (slash) the channel, e.g. 6/stable, 6/candidate, 6/edge. The channels are ordered from the most stable to the least stable, candidate, and edge. More risky channels like edge are always implicitly available. So, if the candidate is listed, you can pull the candidate and edge. When stable is listed, all three are available. 
+
+You can deploy the charm a stand-alone machine or cloud and cloud-like environments, including AWS, Azure, OpenStack and VMWare.
```
Comment (Mehdi-Bendriss) [inline README.md:26]:
```suggestion
You can deploy the charm in a stand-alone machine or in a cloud and cloud-like environments, including AWS, Azure, OpenStack and VMWare.
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+The upper portion of this page describes the Operating System (OS) where the charm can run e.g. 6/edge is compatible and should run in a machine with Ubuntu 22.04 OS.
+
+
+## Security, Bugs and feature request
+
+If you find a bug in this snap or want to request a specific feature, here are the useful links:
```
Comment (Mehdi-Bendriss) [inline README.md:33]:
```suggestion
If you find a bug in this charm or want to request a specific feature, here are the useful links:
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+* Raise issues or feature requests in [Github](https://github.com/canonical/mongos-operator/issues)
+
+* Security issues in the Charmed MongoDB Operator can be reported through [LaunchPad](https://wiki.ubuntu.com/DebuggingSecurity#How%20to%20File). Please do not file GitHub issues about security issues.
+
+* Meet the community and chat with us if there are issues and feature requests in our [Mattermost Channel](https://chat.charmhub.io/charmhub/channels/data-platform)
```
Comment (Mehdi-Bendriss) [inline README.md:39]:
```suggestion
* Meet the community and chat with us if there are issues and feature requests in our [Matrix Room](https://matrix.to/#/#charmhub-data-platform:ubuntu.com)
```

File: README.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
+# Charmed Mongos Documentation
+
+[mongos](https://www.mongodb.com/docs/v6.0/reference/program/mongos/) is a popular NoSQL database application. It stores its data with JSON-like documents creating a flexible user experience with easy-to-use data aggregation for data analytics. In addition, it is a distributed database, so vertical and horizontal scaling come naturally.
```
Comment (Mehdi-Bendriss) [inline README.md:3]: should mongos have the same definition as mongodb (i.e terms like: nosql database ... stores ... documents...)?

File: README.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+Please see the [Juju SDK docs](https://juju.is/docs/sdk) for guidelines on enhancements to this charm following best practice guidelines, and [CONTRIBUTING.md](https://github.com/canonical/mongos-operator/blob/6/edge/CONTRIBUTING.md) for developer guidance.
+
+## License
+
+The Charmed MongoDB Operator is free software, distributed under the Apache Software License, version 2.0. See [LICENSE](https://github.com/canonical/mongos-operator/blob/6/edge/LICENSE) for more information.
+
+The Charmed Mongos Operator is free software, distributed under the Apache Software License, version 2.0. It [installs/operates/depends on] [MongoDB Community Version](https://github.com/mongodb/mongo), which is licensed under the Server Side Public License (SSPL)
+
+See [LICENSE](https://github.com/canonical/mongodb-operator/blob/main/LICENSE) for more information.
```
Comment (Mehdi-Bendriss) [inline README.md:51]:
```suggestion
See [LICENSE](https://github.com/canonical/mongos-operator/blob/main/LICENSE) for more information.
```

**Review by Mehdi-Bendriss — COMMENTED** [review]
Thanks Mia, I left a few comments

**Review by Mehdi-Bendriss — APPROVED** [review]

**Review by dmitry-ratushnyy — APPROVED** [review]

**Pull request merged**